Integrations/JsonWhoIs/JsonWhoIs_documentation.md

```diff
@@ -0,0 +1,112 @@
+## Overview
```

Comment:
documentation needs to be submitted in a new "Documentation Issue" (in etc repository).